pkg/controller/kieapp/constants/constants.go

```diff
@@ -291,6 +291,9 @@ const (
 	SUBCOMPONENT_TYPE_INFRA = "infrastructure"
 
 	DefaultDatagridUsername = "infinispan"
+
+	DefaultTermination = "passthrough"
```

Comment:
Instead of default I would use passthrough as we have enum for them. We still can use passthrough as default termination.
```suggestion
	PassthroughTermination = "passthrough"
```

Reply:
ok